src/coreclr/jit/emitarm64.cpp

```diff
@@ -5198,6 +5209,46 @@ emitter::code_t emitter::emitInsCodeSve(instruction ins, insFormat fmt)
     }
 }
 
+//  For the given 'arrangement' returns the 'elemsize' specified by the SVE vector register arrangement
+//  asserts and returns EA_UNKNOWN if an invalid 'arrangement' value is passed
+//
+/*static*/ emitAttr emitter::optGetSveElemsize(insOpts arrangement)
```

Comment:
I'm thinking this should be merged into `optGetElemsize()` (and that function changed to use a switch), both use  `insOpts`. This would be similar to how `emitDispArrangement()` supports both sve and non-sve.

Reply:
Yea, same with the widen version. I originally did have them merged. On one hand, it makes the API easier to use, but signifies less intent. However, ease of use trumps that.

Reply:
i actually like the separation at least for these methods, where we can catch the instances where we dont end of calling non-scalable opts instead of scalable and vice-versa.